### PR TITLE
replace "Installation Successful" dialog with text in installer window

### PR DESCRIPTION
**Pull request by shuang886**
At least in Sonoma, both the installer and System Settings pop a dialog when the installation completes:

Screenshot 2023-11-23 at 7 16 34 AM Screenshot 2023-11-23 at 7 16 44 AM

which seems distracting. This proposal replaces the installer dialog with some more sedate:

Screenshot 2023-11-23 at 8 56 01 AM

### CODE REVIEW

File: Source/Installer/zh-Hant.lproj/MainMenu.xib [hunk 1]
```diff
@@ -171,7 +171,6 @@ Gw
         <customObject id="494" customClass="AppDelegate">
             <connections>
                 <outlet property="cancelButton" destination="592" id="710"/>
```
Comment (lukhnos) [inline Source/Installer/zh-Hant.lproj/MainMenu.xib:175]: the `@IBOutlet` connection to `actionButton` is missing in the zh-Hant version of the xib.

**Review by lukhnos — CHANGES_REQUESTED** [review]
Good idea, thanks again for the contribution.

This looks good. Just one missing `@IBOutlet` connection in the zh-Hant xib.

File: Source/Installer/zh-Hant.lproj/MainMenu.xib (hunk 1 above)
Comment (shuang886) [reply Source/Installer/zh-Hant.lproj/MainMenu.xib:175]: Oh, rats. Thanks for the catch.

Fixed in https://github.com/shuang886/McBopomofo/commit/353702c4b58529807582ae8c085e2e1fe0b5291a

**Review by lukhnos — APPROVED** [review]
LGTM. Thanks again!

**Pull request merged**